bh/documentation/src/tab-navigation/TabNavigation.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="bh-grid bh-grid--column">
+    <div class="bh-grid__item">
+      <h2>Tab Navigation</h2>
+      <h3>Tab Navigation Example</h3>
+      <example
+        :component="TabNavigationExample"
+        :code="tabNavigationExampleCode" />
+    </div>
+    <div class="bh-grid__item">
+      <api :data="api" />
+    </div>
+  </div>
+</template>
+
+<script>
+import Example from '@/Example';
+import Api from '@/Api';
+import TabNavigationExample from './examples/TabNavigationExample';
```

Comment:
Rename this file `TabNavigationExample` to `TabNavigation`, but keep TabNavigationExample as component name